dom.bs

```diff
@@ -1866,13 +1892,35 @@ them. For instance, if the operation has already completed.
  <li><p>[=Fire an event=] named {{AbortSignal/abort}} at <var>signal</var>.
 </ol>
 
+<p>To <dfn export for=AbortSignal>signal close</dfn>, given an {{AbortSignal}} object
+<var>signal</var>, run these steps:
+
+<ol>
+  <li><p>If <var>signal</var> is [=AbortSignal/aborted=], then return.
+
+  <li><p>If <var>signal</var> is [=AbortSignal/closed=], then return.
+
+  <li><p>Set <var>signal</var>'s [=AbortSignal/closed=] to <var>true</var>.
+
+  <li><p>Remove all of <var>signal</var>'s abort algorithms.
+</ol>
+
+<p class=note>Closing an {{AbortSignal}} enables explicitly indicating that there is no further
+reason to expect the {{AbortController}} to <a for=AbortSignal>signal abort</a> on its
+{{AbortSignal}}, and that the signal can be safely ignored.
+
 <p>A <var>followingSignal</var> (an {{AbortSignal}}) is made to
 <dfn export for=AbortSignal>follow</dfn> a <var>parentSignal</var> (an {{AbortSignal}}) by running
 these steps:
 
 <ol>
  <li><p>If <var>followingSignal</var> is [=AbortSignal/aborted=], then return.
 
+ <li><p>If <var>followingSignal</var> is [=AbortSignal/closed=], then return.
```

Comment:
Ideally, if _followingSignal_ becomes closed later on, we would remove the abort steps added to *parentSignal* so we don't unnecessarily keep *parentSignal* alive.

Not sure how that'll work in practice though. We've already had a similar discussion in https://github.com/whatwg/dom/issues/920#issuecomment-726879818.